workspaces/src/result.rs

```diff
@@ -265,18 +275,18 @@ impl ExecutionOutcome {
         )
     }
 
-    /// Converts this [`ExecutionOutcome`] into a Result type, where the failure is converted
-    /// to an [`anyhow::Error`] object which can be downcasted later.
-    pub fn into_result(self) -> anyhow::Result<ValueOrReceiptId> {
+    /// Converts this [`ExecutionOutcome`] into a Result type to match against whether the
+    /// particular outcome has failed or not.
+    pub fn into_result(self) -> crate::result::Result<ValueOrReceiptId> {
```

Comment:
woops, this was because I had different `Error` types and I kept changing their names as I went on developing. This was left over due to me regex replacing some `Result` with the full path